db/migrate/20190702182118_add_metadata_process_after_to_katello_event.rb

```diff
@@ -0,0 +1,6 @@
+class AddMetadataProcessAfterToKatelloEvent < ActiveRecord::Migration[5.2]
+  def change
+    add_column :katello_events, :metadata, :string
```

Comment:
prefer `text` to `string` here. That seems to be the default for all serializable columns.

Reply:
done